cirq-superstaq/cirq_superstaq/compiler_output.py

```diff
@@ -200,10 +200,12 @@ def read_json(json_dict: dict[str, Any], circuits_is_list: bool) -> CompilerOutp
                 pulses = gss.serialization.deserialize(json_dict["pulses"])
             except Exception as e:
                 s = "s" if circuits_is_list else ""
-                if qiskit.__version__ < "0.24":
+                qiskit_version = qiskit.__version__
+                package_name = "qiskit" if qiskit_version >= "1.0.0" else "qiskit-terra"
+                if qiskit_version < "0.24":
                     warnings.warn(
                         f"Your compiled pulse sequence{s} could not be deserialized, likely "
-                        f"because your Qiskit Terra installation (version {qiskit.__version__}) is "
+                        f"because your {package_name} installation (version {qiskit_version}) is "
```

Comment:
should always just be "Qiskit" - the "terra" name was deprecated well before 1.0.0
```suggestion
                        f"because your Qiskit installation (version {qiskit_version}) is "
```

Reply:
Done in [b8286ee](https://github.com/Infleqtion/client-superstaq/pull/906/commits/b8286ee831933cbacd71e16fb2774a9323ea5eb0)